js/scripts/main.js

```diff
@@ -50,15 +48,38 @@ spanishButton.addEventListener("click", () => {
 	}
 });
 
-/** 
- * If specials keys are pressed, prevent their default action.
- */
+// If specials keys are pressed, prevent their default action.
 document.addEventListener("keydown", (key) => {
 	if (key.altKey || key.ctrlKey || key.metaKey) {
 		key.preventDefault();
 	}
 });
 
+// 1ST APPROACH: 
+// If the window loses focus, pause the game.
+CANVAS.onblur = () => {
```

Comment:
Nice work on this! Thank you for adding this functionality.

Reply:
No problem!